server/game/cards/01_SOR/events/Aggression.ts

```diff
@@ -0,0 +1,42 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { TargetMode, WildcardCardType } from '../../../core/Constants';
+import { UnitCard } from '../../../core/card/CardTypes';
+
+export default class Aggression extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3736081333',
+            internalName: 'aggression',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Aggression modal ability:',
+            immediateEffect: AbilityHelper.immediateEffects.chooseModalEffects({
+                amountOfChoices: 2,
+                choices: () => ({
+                    ['Draw a card']: AbilityHelper.immediateEffects.draw(),
+                    ['Defeat up to 2 upgrades']: AbilityHelper.immediateEffects.selectCard({
+                        mode: TargetMode.UpTo,
```

Comment:
This needs `canChooseNoTargets: true`. Actually could you go to CardTargetResolver and change the behavior so that is the default? I feel like that is almost always the expected behavior in swu. Also, we need to test the choose no targets case